src/main/java/io/github/amerebagatelle/fabricskyboxes/FabricSkyBoxesClient.java

```diff
@@ -29,4 +46,68 @@ public static Logger getLogger() {
         }
         return LOGGER;
     }
+
+    public static void test() {
```

Comment:
I'd like it if this could be separated to a different class.  It doesn't seem quite fitting to have in the client initializer.